client/src/head/index.js

```diff
@@ -1,8 +1,7 @@
 import meta from './meta';
-import mathjax from './mathjax';
 import scripts from './scripts';
```

Comment:
What is this file? Can we consolidate things here?

Reply:
`scripts` is empty now - it used to contain gtag.  I think all the scripts are (as of this PR) in `gatsby-ssr`.

If that's a good place for them, we can just get rid of `scripts` entirely.

Reply:
Yes, please. And thanks for confirming.